CHANGELOG.md

```
@@ -1,3 +1,13 @@
+## Release 7.0.9
+
+#### 27 Aug 2025
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)